tox.ini

```diff
@@ -19,10 +19,13 @@ commands = nosetests
 [testenv:lint]
 basepython = python3
 deps =
-    pycodestyle==2.3.1
+    pycodestyle
+    # need to pin b/c https://github.com/PyCQA/astroid/issues/650
```

Comment:
Ah!

Reply:
yeah...we'll have to delete this once it's fixed upstream :(